src/firmware/main/app/control.c

```diff
@@ -0,0 +1,182 @@
+#include "app/control.h"
+
+#include "physics/physics.h"
+#include "shared/util.h"
+
+/**
+ * Computes the scaling constant to bring the wheel forces to their maximum
```

Comment:
Hmm, I'd prefer to keep specifics out of this, just because it's intended as a high level description, and that sort of detail really does live in the `@return` IMO. Do you have any suggestions for how we could rephrase this while keeping it under a line.

Reply:
Maybe "Computes a scaling constant that can be used to maximize wheel force but also obey physical dynamics" or something similar

Reply:
Ok, slightly longer then one line, but changed to:
```
 * Computes a scaling constant that can be used to maximize wheel force while obeying 
 * physical dynamics
```